R/derive_var_dthcaus.R

```diff
@@ -174,7 +186,7 @@ derive_var_dthcaus <- function(dataset,
       )
```

Comment:
`temp_date = !!sources[[ii]]$date` can be removed as `temp_date` has been created before.

Reply:
I think dropping this line is causing an error - the filter_extreme statement is dropping the temp_date in some instances - I'm guessing because the last line in this function is: select(-starts_with("temp_"))

Reply:
we could recreate temp_date in the same way it's initially created with the impute_dtc function?

Reply:
I would prefer a clean implementation in the next release rather than a work-around for this release.

Reply:
Sure, that makes sense